modules/modal/react/stories/stories.tsx

```diff
@@ -39,7 +69,7 @@ const DefaultModalExample = () => {
   );
 };
 
-const AccessibleModalExample = () => {
+export const AccessibilityTest = () => {
```

Comment:
I'm not sure if this should be a testing only story. Sure, we run tests against this, but the contents of the story were requested by the accessibility team for testing (to make sure our content hiding methods work for mouse users, keyboard users, and users of all kinds of assistive technology).

Stories that are marked as "test only" are still viewable with the URL, but they just won't show up in the navigation.

Reply:
Maybe it doesn't need to be a testing only story, but I think it could live in the `Testing` section (which would require another file